javascript-stringify.js

```diff
@@ -229,6 +229,11 @@
       return PRIMITIVE_TYPES[typeof value](value, indent, next);
     }
 
+    // Handle Error objects.
+    if (value instanceof Error) {
```

Comment:
Maybe checking the `constructor`? It's not great, but might be reasonable. Otherwise, IIRC, there's an ES6 symbol for something around this stuff but the constructor lookup is probably a better place to start.